src/components/media-loader.js

```diff
@@ -65,9 +73,24 @@ AFRAME.registerComponent("media-loader", {
           },
           { once: true }
         );
-        this.el.setAttribute("image-plus", { src: raw, contentType });
+        let blobUrl;
+        if (token) {
+          const imageResponse = await fetch(raw, {
+            method: "GET",
+            headers: { Authorization: `Token ${token}` }
+          });
+          const blob = await imageResponse.blob();
+          blobUrl = window.URL.createObjectURL(blob);
+        }
+        this.el.setAttribute("image-plus", { src: blobUrl || raw, contentType });
         this.el.setAttribute("position-at-box-shape-border", { target: ".delete-button", dirs: ["forward", "back"] });
-      } else if (contentType.startsWith("model/gltf") || url.endsWith(".gltf") || url.endsWith(".glb")) {
+      } else if (
+        contentType.indexOf("application/octet-stream") !== -1 ||
```

Comment:
Not super in love with gltf's just being the fallback for any octet stream or zip file...